source/manual/setting-up-new-rails-app.html.md

```diff
@@ -16,137 +16,174 @@ parent: "/manual.html"
+[auto-config]: https://docs.publishing.service.gov.uk/manual/configure-github-repo.html#auto-configuration
+[app-list]: https://docs.publishing.service.gov.uk/#applications
 
-## Familiarise yourself with the conventions
+## Before you start
 
-We have documented [conventions for Rails
-applications](/manual/conventions-for-rails-applications.html) and
-[conventions for naming applications](/manual/naming.html) in GOV.UK.
+Before you create a Rails app, you must complete all steps in the [GOV.UK developer get started documentation](get-started).
 
-## Bootstrapping a new application
+## Create a new Rails app
 
-To create a rails app, run the following (skip uncommon stuff).
+1. To create a Rails app, go to the development space on your local machine and run the following in the command line:
 
-```sh
-rails new myapp --skip-javascript --skip-test --skip-bundle --skip-spring --skip-action-cable --skip-action-mailer --skip-active-storage
-```
+    ```sh
+    rails new myapp --skip-javascript --skip-test --skip-bundle --skip-spring --skip-action-cable --skip-action-mailer --skip-active-storage
+    ```
 
-Replace the Gemfile with the gems you need. Here is an example.
+    The `--skip` commands exclude unneeded Rails plugins.
 
-```rb
-source "https://rubygems.org"
+1. Replace the Gemfile with the gems you need. The following code shows an example.
 
-gem "rails", "6.0.3.4"
+    ```rb
+    source "https://rubygems.org"
 
-gem "bootsnap",
-gem "gds-api-adapters"
-gem "gds-sso"
-gem "govuk_app_config"
-gem "govuk_publishing_components"
-gem "pg"
-gem "plek"
-gem "uglifier"
+    gem "rails", "6.0.3.4"
 
-group :development do
-  gem "listen"
-end
+    gem "bootsnap",
+    gem "gds-api-adapters"
+    gem "gds-sso"
+    gem "govuk_app_config"
+    gem "govuk_publishing_components"
+    gem "pg"
+    gem "plek"
+    gem "uglifier"
 
-group :test do
-  gem "simplecov"
-end
+    group :development do
+      gem "listen"
+    end
 
-group :development, :test do
-  gem "byebug"
-  gem "govuk_test"
-  gem "rspec-rails"
-  gem "rubocop-govuk"
-end
-```
+    group :test do
+      gem "simplecov"
+    end
 
-Run `bundle && rails g rspec:install` and replace `spec/*helper.rb`.
+    group :development, :test do
+      gem "byebug"
+      gem "govuk_test"
+      gem "rspec-rails"
+      gem "rubocop-govuk"
+    end
+    ```
 
-```sh
-rm spec/rails_helper.rb
-```
+1. Run `bundle && rails g rspec:install` to install all gems specified in the Gemfile and generate files used for testing.
 
-```rb
-## spec/spec_helper.rb
-ENV["RAILS_ENV"] ||= "test"
+1. Delete `spec/rails_helper.rb`:
 
-require "simplecov"
-SimpleCov.start "rails"
+    ```sh
+    rm spec/rails_helper.rb
+    ```
 
-require File.expand_path("../../config/environment", __FILE__)
-require "rspec/rails"
+    Open `spec_helper.rb` and replace its contents with the following:
 
-Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
-GovukTest.configure
+    ```rb
+    ## spec/spec_helper.rb
```

Comment:
I think this can be changed to:

```
    Open `spec/spec_helper.rb` and replace its contents with the following:

    ```rb
```

The file is `spec/spec_helper.rb`. and the first line of the example code is just repeating the filename.